.github/workflows/sqlsmith.yaml

```diff
@@ -8,13 +8,24 @@ on:
     branches:
       - sqlsmith
 jobs:
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      pg14_latest: ${{ steps.setter.outputs.PG14_LATEST }}
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v2
+    - name: Build matrix
+      id: setter
+      run: python .github/gh_config_reader.py
+
   regress:
     name: SQLsmith PG${{ matrix.pg }}
     runs-on: ${{ matrix.os }}
     strategy:
```

Comment:
Most other GitHub actions that are using `gh_config_reader.py` have a dependency on the new `config` job (`needs: config`). From my point of view, this would also make sense for the `regress` job.